niworkflows/utils/misc.py

```diff
@@ -299,19 +299,28 @@ def check_valid_fs_license():
         FreeSurfer successfully executed (valid license)
 
     """
+    import os
     from pathlib import Path
     import subprocess as sp
     from tempfile import TemporaryDirectory
     from pkg_resources import resource_filename
 
+    env = os.environ.copy()
+    if lic is not None:
+        import warnings
+        warnings.warn("The license argument has been deprecated. Please set the environment "
```

Comment:
this is most likely too long but let's note the quick removal.

```suggestion
        warnings.warn("The license argument has been deprecated and will be removed in 1.3.0. Please set the environment "
```